tests/test_datasets/test_dataset.py

```diff
@@ -234,61 +212,48 @@ def setUp(self):
         self.sparse_dataset = openml.datasets.get_dataset(4136, download_data=False)
 
     def test_get_sparse_dataset_with_target(self):
-        X, y = self.sparse_dataset.get_data(
+        X, y, _, attribute_names = self.sparse_dataset.get_data(
             dataset_format='array', target="class"
         )
         self.assertTrue(sparse.issparse(X))
         self.assertEqual(X.dtype, np.float32)
         self.assertIsInstance(y, np.ndarray)
         self.assertIn(y.dtype, [np.int32, np.int64])
         self.assertEqual(X.shape, (600, 20000))
-        X, y, attribute_names = self.sparse_dataset.get_data(
-            dataset_format='array',
-            target="class",
-            return_attribute_names=True,
-        )
         self.assertTrue(sparse.issparse(X))
```

Comment:
I assume this check is no longer necessary?

Reply:
There is no check here. This is just a call to retrieve attribute names, which is now always done by default anyway.